src/constants/index.ts

```diff
@@ -1 +1,3 @@
 export { default as PATHS } from './paths';
+export { default as COLORS } from './colors';
+export { default as LEVELS } from './pollution';
```

Comment:
```suggestion
export { default as POLLUTION } from './pollution';
```
I would name this variable the same as the file from where you export these constants.

Reply:
Of course it should be the same, but you wanted it to be called `pollution.ts` and it was not intuitively giving information that this list is containing the pollution levels scale.

Reply:
Anyway, changed.